Description: Fix format string vulnarabilities.
Author: Fabian Greffrath <fabian+debian@greffrath.com>

--- a/xvpbm.c
+++ b/xvpbm.c
@@ -885,8 +885,8 @@ int WritePBM(fp,pic,ptype,w,h,rmap,gmap,
 	  }
 	}
 	else {
-	  if (*pix) fprintf(fp,str1);
-	       else fprintf(fp,str0);
+	  if (*pix) fprintf(fp,"%s",str1);
+	       else fprintf(fp,"%s",str0);
 	  len+=2;
 	  if (len>68) { fprintf(fp,"\n"); len=0; }
 	}
